dev-docs/npn-and-go-upgrades.md

```diff
@@ -0,0 +1,45 @@
+Taskcluster tries to use the LTS release of Node.js (see
+[release schedule](https://nodejs.org/en/about/releases/)), and the latest
+release of [Go](https://go.dev/). It may be a good idea to update these
+after a Taskcluster release, to get some time with the update in
+development environments before deploying new versions.
+
+# Updating node
+
+Dependabot will open PRs for Node.js, but these will often fail since
+additional steps are needed.
+
+To update the node version:
+
+* Install the new node version, such as `nvm install 16.13.2`
```

Comment:
`nvm install --lts` will also install the latest LTS, wonder if that's worth mentioning here? Up to you.